src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return String.valueOf(Color.values()[index]);
```

Comment:
```suggestion
        return Color.values()[index].name();
```

Reply:
not fixed
![image](https://user-images.githubusercontent.com/20326858/227518845-4dfd87c2-0141-447c-8748-25d67bbf5308.png)